index.html

```diff
@@ -15,7 +19,9 @@ <h1 class="title">
 			<div class="logo">
 				<img src="static/htf-logo.png" alt="Hack the Fog Logo">
 			</div>
-			<h2>San Francisco's first and awesome-est high school hackathon</h2>
+			<h2>San Francisco's first and best high school hackathon</h2>
+			<center><h4>Hosted by Lowell Byte Lab</h4>
```

Comment:
For organization, I was thinking maybe we use "text-align: center" CSS instead of center tags, but this works too.

Reply:
I just remembered that I already have text-align: center for all the text in the info tabs... I'll take these tags out for organization. sorry ^^;

Reply:
Thats fine when I looked at the code there were a lot of spaces.  I thought the spaces was what made it centered.